src/Sarif.Viewer.VisualStudio/OpenLogFileCommands.cs

```diff
@@ -298,7 +298,7 @@ string ConvertSarifProtocol(string inputUrl)
             return newUrl;
         }
 
-        bool TryDownloadFile(string inputUrl, out string downloadedFilePath)
+        private bool TryDownloadFile(string inputUrl, out string downloadedFilePath)
```

Comment:
static?

Reply:
yes

---
In reply to: [519462625](https://github.com/microsoft/sarif-visualstudio-extension/pull/259#discussion_r519462625) [](ancestors = 519462625)